inlong-agent/conf/log4j2.xml

```diff
@@ -37,10 +38,13 @@
         <property name="error_filePattern">${basePath}/error-%d{yyyy-MM-dd}-%i.log.gz</property>
         <property name="error_max">10</property>
         <property name="console_print_level">DEBUG</property>
+        <property name="index_max">10</property>
         <property name="index_fileName">${basePath}/index.log</property>
         <property name="index_filePattern">${basePath}/index-%d{yyyy-MM-dd}-%i.log.gz</property>
+        <property name="monitors_max">10</property>
         <property name="monitors_fileName">${basePath}/monitors.log</property>
         <property name="monitors_filePattern">${basePath}/monitors-%d{yyyy-MM-dd}-%i.log.gz</property>
+        <property name="last_modify_time">10d</property>
```

Comment:
Can increase the default number of saved days to 15 days?

Reply:
resolved